Examples/Immense.RemoteControl.Examples.LinuxDesktopExample/BrandingProvider.cs

```diff
@@ -10,20 +10,23 @@ internal class BrandingProvider : IBrandingProvider
         Product = "Immy Remote"
     };
 
-    public BrandingProvider()
-    {
-        using var mrs = typeof(BrandingProvider).Assembly.GetManifestResourceStream("Immense.RemoteControl.Examples.LinuxDesktopExample.ImmyBot.png");
-        using var ms = new MemoryStream();
-        mrs!.CopyTo(ms);
 
-        _brandingInfo.Icon = ms.ToArray();
-    }
+    public BrandingInfoBase CurrentBranding => _brandingInfo;
 
     public Task<BrandingInfoBase> GetBrandingInfo()
     {
         return Task.FromResult(_brandingInfo);
     }
 
+    public async Task Initialize()
+    {
+        using var mrs = typeof(BrandingProvider).Assembly.GetManifestResourceStream("Immense.RemoteControl.Examples.LinuxDesktopExample.ImmyBot.png");
```

Comment:
lol, I read this a mrs and mr, as if this variable was female and the one below was male.

I then chuckled again when I read `await mrs!.CopyToAsync(ms);` , doesn't even make sense but I still thought it was funny